.github/workflows/deploy.yml

```diff
@@ -18,6 +18,21 @@ jobs:
       - run: npm i -g pnpm@7.1.9
       - run: pnpm install --frozen-lockfile
       - run: pnpm build
+
+      - name: Deploy usedapp-docs.netlify.app
+        uses: nwtgck/actions-netlify@v1.2
+        with:
+          enable-commit-comment: true
+          enable-pull-request-comment: true
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          overwrites-pull-request-comment: true
+          production-deploy: ${{ github.event_name != 'pull_request' }}
+          publish-dir: packages/docs/build
+        env:
+          NETLIFY_AUTH_TOKEN: ${{ secrets.NETLIFY_AUTH_TOKEN }}
+          NETLIFY_SITE_ID: ${{ secrets.NETLIFY_DOCS_SITE_ID }}
+        if: ${{ github.event_name == 'pull_request' }} # Deployed separately to production on release
```

Comment:
I feel like `${{ (github.event_name == 'push') && (github.ref_name != 'master') }}` would make more sense but not sure about this one